### PR TITLE
updated code to python3; all tests pass

### PR DESCRIPTION
**Pull request by joshuago78**

### CODE REVIEW

**Comment by kirschbombe** [comment]
I will test this tomorrow morning, @joshuago78.

File: README.md [hunk 1]
```diff
@@ -9,23 +9,22 @@ NOID-mint Python Package
...
 
 5. Activate the virtual environment
-	$ source venv/bin/activate
+	$ source ENV/bin/activate
 
-6. Confirm the python version is 2.7 by running 
+6. Confirm the python version is 3.x by running 
 	$ python --version
 	
 7. Navigate down into the NOID-mint directory
```
Comment (kirschbombe) [inline README.md:23]: 7. Navigate down into the SLDP_ARK_mint directory

File: README.md [hunk 2]
```diff
@@ -9,23 +9,22 @@ NOID-mint Python Package
...
 	$ cd NOID-mint
```
Comment (kirschbombe) [inline README.md:24]: $ cd SLDP_ARK_mint

**Comment by kirschbombe** [comment]
Hi, @joshuago78. Set up and installation went fine, but I ran into an error running the `sinai_work_page_ark.py` script:
```
(arks3) kirschbombe@YRL-ML-DLP003 SLDP_ARK_mint % python sinai_work_page_ark.py 
Traceback (most recent call last):
  File "sinai_work_page_ark.py", line 31, in <module>
    directory = (str(raw_input('File directory:')).strip())+'/'
NameError: name 'raw_input' is not defined
```

**Comment by joshuago78** [comment]
OK, I have removed the deprecated `raw_input()` methods. I have also added the other two requirements: using Shelfmark and looking for works*.csv files

**Comment by kirschbombe** [comment]
thanks! I will test this in the morning (or tonight if I'm not too tired).

**Comment by kirschbombe** [comment]
I tested again today, but received a lot more errors. I put them in a gist here: https://gist.github.com/kirschbombe/fe8050dfeafadbdd2b8976d7d4368788.js

**Comment by joshuago78** [comment]
Closing this PR. Andy and Zoe will take a look at the changes I made and begin a new branch/PR

**Pull request closed without merging**